.vscode/settings.json

```diff
@@ -53,5 +53,17 @@
     "mutex": "cpp",
     "ratio": "cpp",
     "thread": "cpp"
-  }
+  },
+  "workbench.colorCustomizations": {

```

Comment:
I think it's not needed.

Reply:
Yeah! that is Peacock's doing, I'll remove it.